addons/web/static/tests/views/form_tests.js

```diff
@@ -4472,6 +4472,52 @@ QUnit.module('Views', {
         form.destroy();
     });
 
+    QUnit.test('navigation with tab key selects a value in form view', async function (assert) {
```

Comment:
did you check that other fields weren't already tested?

Reply:
@aab-odoo I didn't find any test where select value is tested, so I added for all other widget as well